### PR TITLE
feat: [NUI-37] styles-package

### PR DESCRIPTION
**Pull request by gabssanto**
Added custom Typography, custom theme, custom border-radius, custom shadows and custom palette

### CODE REVIEW

File: packages/styles/src/lib/Theme/palette.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+export const palette: PaletteOptions = {
+  grayScale: {
+    blackHole: colors['blackHole'],
+    eclipse: colors['eclipse'],
+    penumbra: colors['penumbra'],
+    spaceStation: colors['spaceStation'],
+    fullMoon: colors['fullMoon'],
+    newMoon: colors['newMoon'],
+    halo: colors['halo'],
+    superNova: colors['supernova'],
```
Comment (mgonc-verta) [inline packages/styles/src/lib/Theme/palette.ts:43]: supernova is a single word

File: packages/styles/src/lib/styles.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { render } from '@testing-library/react';
+
+import Styles from './styles';
+
+describe('Styles', () => {
```
Comment (mgonc-verta) [inline packages/styles/src/lib/styles.spec.tsx:5]: we can remove this

File: packages/styles/src/lib/styles.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import styled from '@emotion/styled';
+
+/* eslint-disable-next-line */
+export interface StylesProps {}
+
+const StyledStyles = styled.div`
+  color: pink;
+`;
+
```
Comment (mgonc-verta) [inline packages/styles/src/lib/styles.tsx:9]: we can remove this

File: packages/styles/src/lib/Theme/index.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    ...others
+  } = options ?? {};
+  return MUICreateTheme({
+    palette: {
+      ...themePalette,
+      ...palette,
+    },
+    spacing: (factor: Spacings): Length => `${Number(factor) * 0.5}rem`,
+    shadows: themeShadows,
+    components: {
```
Comment (mgonc-verta) [inline packages/styles/src/lib/Theme/index.ts:30]: maybe we can create also the prop customShadow that serves the shadow token object.

**Review by mgonc — APPROVED** [review]

**Pull request merged**